client/analytics/report/revenue/index.js

```diff
@@ -229,28 +297,10 @@ class RevenueReport extends Component {
 				/>
 				<ReportFilters query={ query } path={ path } />
 
-				<SummaryList>
-					<SummaryNumber
-						value={ formatCurrency( totals.gross_revenue ) }
-						label={ __( 'Gross Revenue', 'wc-admin' ) }
-						delta={ 29 }
-					/>
-					<SummaryNumber
-						value={ formatCurrency( totals.refunds ) }
-						label={ __( 'Refunds', 'wc-admin' ) }
-						delta={ -10 }
-						selected
-					/>
-					<SummaryNumber
-						value={ formatCurrency( totals.coupons ) }
-						label={ __( 'Coupons', 'wc-admin' ) }
-						delta={ 15 }
-					/>
-					<SummaryNumber
-						value={ formatCurrency( totals.taxes ) }
-						label={ __( 'Taxes', 'wc-admin' ) }
-					/>
-				</SummaryList>
+				{ this.getChartSummaryNumbers() }
+				<Card title="">
```

Comment:
I think we should just drop the card wrapper here for now. The `Chart` already contains the header, and this just adds an extra blank box.